### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by santitobon9**
Merging my implementation of a simulated draft.

### CODE REVIEW

File: data/league.txt [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+Dustin Le,RIP KOBE,8
+Mack Cooper,Oops All Kneecapped,7
+Casey Kunzat,Brooklyn Nets Finals Bound,11
+Charles Clasuen,Taco Tuesday with Vuccie Mane,1
+Nick Cartier,Day to Day,2
+Trevor Liberty,Process Truster T,4
+Cody Jeffries,The North Couve,12
+Peter Crljenica,Petes HouseofRUI,10
+Jason Lee,To Be Determined,3
+Bethanie Jeffries,Bora Bora Beth,5
```
Comment (santitobon9) [inline data/league.txt:13]: Added draft ordering

File: src/draft.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    return
+
+
+player_pool = pd.read_csv("data/espn_data_set.csv")
+my_league, draft_order, num_of_rounds = lg.read_league_file(
+    "data/league.txt", player_pool
+)
+already_drafted = {}
+draft = draft(my_league, num_of_rounds, player_pool, draft_order, already_drafted)
+simulate_draft(draft)
```
Comment (santitobon9) [inline src/draft.py:88]: Simulates a fantasy draft

File: src/draft_analysis.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            optimizer_analysis2(roster, player_pool)
+        elif choice == "5":
+            ex = True
+
+    return
+
+
+df = pd.read_csv("data/espn_data_set.csv")
+roster = sfb.mean_player(df)
+analyze_roster(roster, df)
```
Comment (santitobon9) [inline src/draft_analysis.py:104]: Separate from the simulated draft. This allows the user to analysis individual players if they were on their team and see the net change with those players.

File: src/league.py [hunk 4]
```diff
@@ -1,59 +1,57 @@
...
+
+def read_league_file(file_name, df):
+    map_of_league = {}
+    draft_order = {}
+    with open(file_name) as file:
+        for i, f in enumerate(file):
+            lines = f.strip().split("\n")
+            for l in lines:
+                seg = l.split(",")
+                # print(i)
```
Comment (mackkcooper) [inline src/league.py]:
```suggestion
```

**Review by mackkcooper — APPROVED** [review]
This needs more comments and tests, but I won't block on that.

**Pull request merged**